data/events.xml

```diff
@@ -1,5 +1,226 @@
+        <endDate>2022-02-06</endDate>
+        <location>Brussels, Belgium</location>
+        <speaker>Nicola Corti,  Martin Bonnin, Marco Gomiero, Holger Steinhauer, Julien Salvi</speaker>
+        <title>Fosdem 2022</title>
+        <subject>Kotlin DevRoom Welcoming Remarks</subject>
+        <url>https://fosdem.org/2022/schedule/event/kotlin_devroom_welcome/</url>
+        <description>
+            <![CDATA[
+<p>Welcoming participants to the virtual edition of the Kotlin DevRoom @ FOSDEM<p>
+]]>
+       </description>
+    </event>
+
+    <event>
+        <lang>en</lang>
+        <startDate>2022-02-06</startDate>
+        <endDate>2022-02-06</endDate>
+        <location>Brussels, Belgium</location>
+        <speaker>Anton Arhipov</speaker>
+        <title>Fosdem 2022</title>
+        <subject>The State of Kotlin</subject>
+        <url>https://fosdem.org/2022/schedule/event/kotlin_devroom_welcome/</url>
+        <description>
+            <![CDATA[
+<p>In 2021 Kotlin turned 10 and the team delivered 2 major releases (1.5.0 and 1.6.0) and a few minors. The main themes of those releases were Java interoperability and consistency of the language features. Meanwhile, Kotlin/JS has been promoted to Beta and KMM is on its path to reaching Beta status as well. However, one long-running project in Kotlin is the development of the new compiler, K2, which should set a solid foundation for the language and the ecosystem for the years ahead.<p>
+<p>This session is an overview of the latest developments in Kotlin: the language, tools, compiler, and the ecosystem around it. We're also going to take a look at Kotlin's roadmap to see where are we heading. Come and learn what's new and exciting!<p>
+]]>
+       </description>
+    </event>
+
+    <event>
+        <lang>en</lang>
+        <startDate>2022-02-06</startDate>
+        <endDate>2022-02-06</endDate>
+        <location>Brussels, Belgium</location>
+        <speaker>Anton Arhipov</speaker>
+        <title>Fosdem 2022</title>
+        <subject>The beauty of Kotlin typing system</subject>
+        <url>https://fosdem.org/2022/schedule/event/the_beauty_of_kotlin_typing_system/</url>
+        <description>
+            <![CDATA[
+<p>The Kotlin typing system is amazingly designed. It was designed and constantly improved for years, and it was built based on experiences from other typing systems. As a result it gives us very comfortable nullability support, type inference, universal guards, and much more. In this presentation, we’ll dive deep into Kotlin typing systems and how small design choices lead to a better programming experience.<p>
+]]>
+       </description>
+   </event>
+
+   <event>
+        <lang>en</lang>
+        <startDate>2022-02-06</startDate>
+        <endDate>2022-02-06</endDate>
+        <location>Brussels, Belgium</location>
+        <speaker>Romain Boisselle</speaker>
+        <title>Fosdem 2022</title>
+        <subject>The beauty of Kotlin typing system</subject>
```

Comment:
@TinaNEs seems the name is wrong

Reply:
i made a mistake in the title. 
Now it is fixed.